app/views/defence_requests/_form.html.erb

```diff
@@ -0,0 +1,119 @@
+<% if @defence_request.new_record? %>
+  <%= render partial: 'solicitor_search', locals: {object: @defence_request} %>
```

Comment:
Should be able to remove the `:partial` and `:locals` keys since these are the defaults since Rails 3, e.g:

``` erb
<%= render 'solicitor_search', object: @defence_request %>
```